tests/models/wav2vec2/test_modeling_wav2vec2.py

```diff
@@ -1995,3 +1998,50 @@ def run_model(lang):
 
         for lang in LANG_MAP.keys():
             assert run_model(lang) == TRANSCRIPTIONS[lang]
+
+    @require_flash_attn
+    @require_torch_gpu
+    @pytest.mark.flash_attn_test
+    def test_flash_attn_2_inference_equivalence(self):
```

Comment:
Currently, this test looks more like a fast test in terms of what it checks (that the logits from fa2 match those from eager). I would move it into the fast tests and use the dummy models as done here: https://github.com/huggingface/transformers/blob/9b5a6450d481b0f02834684ffd8b3ba4cbbd6fe0/tests/models/whisper/test_modeling_whisper.py#L891 

Note that if we load two full-size models and do two forward passes, the slow test becomes un-necessarily heavy. Instead, we can use the transcription from the "eager" model as the expected transcription (or ground truth), and then simply check that the transcription from the "fa2" model matches what we expect.

More specifically, I would copy the existing test from: https://github.com/huggingface/transformers/blob/7afade208695f1a25dcc9656d553d51ace67264e/tests/models/wav2vec2/test_modeling_wav2vec2.py#L1481

But update it to use `attn_implementation="flash_attention_2"`, confirming that the transcription result is the same as the current `EXPECTED_TRANSCRIPTIONS`

=> this is good enough to confirm the model works with FA2 with inference

Reply:
Actually, `test_flash_attn_2_inference_equivalence` is now computed in [test_modelling_common](https://github.com/huggingface/transformers/blob/8c00b53eb01051e93942e99912455883b7ea47b2/tests/test_modeling_common.py#L3248), so we can remove it here.